src/events.ts

```diff
@@ -80,6 +80,25 @@ export async function sendEvent(event, to) {
   }
 }
 
+async function sendEventToDiscordSubscribers(event: string, proposalId: string) {
+  try {
+    const url = `${discordBotUrl}/api/event-to-subscribers`;
+    const params = {
+      event,
+      proposalId
+    };
+    await fetch(url, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ params })
+    });
+
+    return console.log('[events] Notify Discord subscribers success');
+  } catch (error) {
+    console.log("[events] Discord API Failure", error);
+  }
+}
```

Comment:
I feel like we should include this `${discordBotUrl}/api/event-to-subscribers` in our webhooks table. sending events that way will be more secure.

Reply:
Could you potentially give me a reference link for that? :pray: 

I can also implement authorization for the post request on the `discord-bot` side, then we can store i.e. a bearer token in the env.